modules/ROOT/pages/push/proc_preparing-ups-for-use-with-ios.adoc

```diff
@@ -0,0 +1,109 @@
++
+From the *KeyChain Access* menu, there is now a *private* key:
++
+image::apns-keychain-prtkey.png[private]
+
+6. Open the link:https://developer.apple.com/account/overview.action[Provisioning Portal] and log in with your Apple developer account.
+
+7. Click the *Identifiers* link in order to create a new App ID.
+
+8. In the formular, give the *App ID* a _descriptive_ name, for example, *My first AeroGear Push App*.
+
+9. Ensure that the *Push Notifications* checkbox is _selected_:
++
+image::apns-appid-1.png[Give the App ID a name]
+
+10. Enter *YOUR* bundle ID in the _Bundle ID_ field. In this example we choose _org.aerogear.PushTest_, however you *must* use your own ID.
++
+NOTE: The _Bundle ID_ has to match the one from the actual iOS application that you are building.
++
+image::apns-appid-2.png[Apply the Bundle ID from our App]
+
+11. On the next screen, confirm your new App ID, and ensure that the _Push Notifications_ option is _enabled_.
+
+12. Click the _Submit_ button to complete this process.
+
+13. On the next screen, click on the _newly created_ App ID.
+
+14. The following screen appears:
++
+image::apns-appid-3.png[Overview]
+
+15. Click the Edit button and scroll down to the Push Notifications section.
+
+16. Click the _Create Certificate..._ button on the _Development SSL Certificate_ section to generate a _Development Certificate_:
++
+image::apns-sslcert.png[SSL Certificate]
+
+17. Perform an upload of the *CSR* and click the _Generate_ button.
+
+18. Once complete, click the _Edit_ button and scroll down to the Push Notifications section.
++
+Notice that the _Development SSL Certificate_ section is filled with the required details:
++
+image::apns-sslcert-prod.png[SSL Certificate Production]
+
+19. Create the _Production_ certificate by following the same steps however this time click the 'Create Certificate' button on the _Production SSL Certificate_ section.
+
+20. On the overview of the *App ID*, click the _Edit_ button and scroll down to the Push Notifications section.
+
+21. Now that both Certificates are generated, download both to your computer.
+
+22. Double click the _aps_development.cer_ file (this is the _Development SSL Certificate_) and it displays in the *KeyChain Access* tool:
++
+image::apns-pushcert.png[APNs certificate]
+
+23. Export the certificate/private key pair to the *.p12* (_Personal Information Exchange_).
++ NOTE: Ensure to store the _passphrase_ in a secure location.
++
+image::apns-pushcert-export.png[Export to .p12 format]
+
+24. Use the steps above to download the _Production SSL Certificate_.
+
+25. Double click the _aps_production.cer_ file (this is the _Production SSL Certificate_) and it displays in the *KeyChain Access* tool.
+
+26. Export the *.p12* (_Personal Information Exchange_).
++ NOTE: Ensure to store the _passphrase_ in a secure location.
+
+27. Next, you need to create an link:#provisioning-profiles[Apple Provisioning Profile].
+
+[discrete]
+== Additional Resources
```

Comment:
This can be removed, if you can think of anything that might be useful for this section, do please add it

Reply:
I've reworked this